test/Microsoft.Azure.WebJobs.Host.FunctionalTests/FunctionalTest.cs

```diff
@@ -196,15 +196,15 @@ internal static class FunctionalTest
                 new ExpectManualCompletionFunctionInstanceLogger<TResult>(taskSource, ignoreFailureFunctions), extensions);
         }
 
-        private static IServiceProvider CreateServiceProvider<TResult>(IStorageAccount storageAccount, Type programType,
+        private static JobHostConfiguration CreateServiceProvider<TResult>(IStorageAccount storageAccount, Type programType,
             IExtensionTypeLocator extensionTypeLocator, TaskCompletionSource<TResult> taskSource,
             IFunctionInstanceLogger functionInstanceLogger, IExtensionRegistry extensions = null)
         {
             return CreateServiceProvider<TResult>(storageAccount, programType, extensionTypeLocator,
                 DefaultJobActivator.Instance, taskSource, functionInstanceLogger, extensions);
         }
 
-        private static IServiceProvider CreateServiceProvider<TResult>(IStorageAccount storageAccount, Type programType,
+        private static JobHostConfiguration CreateServiceProvider<TResult>(IStorageAccount storageAccount, Type programType,
```

Comment:
Can we rename these methods to CreateConfiguration or CreateTestConfiguration since that is what they now do?